### DISCUSSION TITLE
Multiple mesh transformations in libigl

### DISCUSSION THREAD

**Discussion by xander-Ito-Low** [libigl/libigl#2167] [Q&A]
Hi, are transformations on independent meshes possible in the scene in libigl? I have been reading on other threads that transformations are applied before the meshes are rendered but was unsure if this has changed. I was hoping to render a scene and then apply transformations on objects and then re-render the scene.

Any advice would be helpful. Thank you.

**Comment by alecjacobson** [comment]
currently you'd do this outside of the viewer and use `viewer.data().set_vertices()` to update the positions.

**Reply by xander-Ito-Low** [reply]
Thank you for the the help! I will also investigate the polyscope library because it seems to have support for a more complex viewer and it can be integrated with the geometry functionality from libigl.
